src/api/core/organizations.rs

```diff
@@ -321,41 +321,49 @@ async fn get_org_collections_details(org_id: &str, headers: ManagerHeadersLoose,
         None => err!("User is not part of organization"),
     };
 
+    // get all collection memberships for the current organization
     let coll_users = CollectionUser::find_by_organization(org_id, &mut conn).await;
 
+    // check if current user has full access to the organization (either directly or via any group)
+    let has_full_access_via_group =
+        CONFIG.org_groups_enabled() && GroupUser::has_full_access_by_member(org_id, &user_org.uuid, &mut conn).await;
+    let has_full_access_to_org = user_org.access_all || has_full_access_via_group;
```

Comment:
This can probably be a bit more efficient.
Currently there is a query done to check if the user has full access via the group.
But, that is not needed when the user already has full access.

Probably this should work (not tested):
```suggestion
    let has_full_access_to_org = user_org.access_all || (CONFIG.org_groups_enabled() && GroupUser::has_full_access_by_member(org_id, &user_org.uuid, &mut conn).await);
```